build_tools/cmake/test_riscv.sh

```diff
@@ -99,6 +99,8 @@ test_exclude_args+=(
   "iree/tests/e2e/matmul/e2e_matmul_direct_f32_small_llvm-cpu_local-task"
   "iree/tests/e2e/matmul/e2e_matmul_direct_f32_small_no_padding_llvm-cpu_local-task"
   "iree/tests/e2e/regression/check_regression_llvm-cpu_strided_slice.mlir"
+  # TODO(#15540): Enable the tests.
+  "iree/tests/e2e/linalg/check_llvm-cpu_local-task_f32_to_i4.mlir"
```

Comment:
@pzread @ScottTodd How do I disable the compilation for risc-v targets?

Reply:
Try the `noriscv` tag/label, as in https://github.com/openxla/iree/blob/2f9a1e173237b0948ef4f2bd3e03c85856d1be27/tests/e2e/matmul/BUILD.bazel#L39-L42

That should filter at _configure time_ (prior to _build time_), while this change to `test_riscv.sh` just filters at _test time_
https://github.com/openxla/iree/blob/fc441856b26be7691ed9406a7c8617befd07b036/build_tools/cmake/iree_check_test.cmake#L9-L11 https://github.com/openxla/iree/blob/fc441856b26be7691ed9406a7c8617befd07b036/build_tools/cmake/iree_check_test.cmake#L68-L71

Reply:
oh cool, I did not know the tag. Let me give it a shot. Thanks!!

Reply:
btw: bazel_to_cmake should correctly propagate the BUILD file tag to the CMake file (your commit just modified the Bazel file)

Reply:
ooops.. good catch. I forgot to run the script before commit.

Reply:
It is still compiled for risc-v targets. Any thoughts?

Reply:
The failing compile is `check_winograd_llvm-cpu_local-task_f32_to_i4.mlir_module.vmfb` ([logs here](https://github.com/openxla/iree/actions/runs/6830324343/job/18578279271?pr=15539#step:7:1028))

The `check_winograd_llvm-cpu_local-task` suite is missing the label.

However, looking closer... I think you only actually want to skip the new `f32_to_i4` tests, not all of the tests in `LLVM_SRCS`. You could add a separate group like `LLVM_I4_SRCS` and add new test suites using those sources, then only apply the label to those suites.

Reply:
okay... the winograd test suite should only contain `conv2d.mlir`. I believe that the preprocessing pass is only applied to the test. I can fix it...

Create a test suite for i4_store for now.